README.md

```diff
@@ -0,0 +1,134 @@
+epipearl
+===============================
+
+Python client for [epiphan-pearl][pearl]. It provides a client interface for
+epiphan-pearl [http api][pearl-http-api], and some of its web ui configuration
+interface, and DCE custom web ui configuration as well.
+
+Current version of Epipearl is tested against _v3.15.3f_ firmware.
+
+This software shoud be considered alpha, therefore likely to change/break in the near future.
+
+
+install
+-------
+
+    pip install epipearl
+
+
+
+example usage to get and set channels parameters
+------------------------------------------------
+
+    from epipearl import Epipearl
+    client = Epipearl( "http://epiphan_pearl_address", "admin", "secret_password" )
+    
+    # to get the type of stream being published and frame size for channel 1
+    response = client.get_params( channel='1', params={'publish_type':'', 'framesize':''})
+    print "publish_type is %s" % response['publish_type']
+    print "framsize is %s" % response['framesize']
+    
+    # to start recording on recorder 2
+    response = client.set_params( channel='m2', params={'rec_enabled': 'on'})
+    if response:
+        print "recorder 2 set to start recording"
+
+
+
+example usage of web ui methods
+------------------------------------------------
+
+Besides the HTTP API calls `get_params()` and `set_params()`, epipearl exposes
+some web ui HTML form posts as python calls.
+
+For example, if you need to create a new channel:
+
+    # create the client
+    import requests
```

Comment:
Or maybe this is a situation where epipearl _ought_ to have it's own exception type that wraps `HTTPError`.